### PR TITLE
Add more food and drinks

### PR DESCRIPTION
**Pull request by AtomicFox556**
#### Summary
Content "Add more food and drinks"

#### Purpose of change
Expand the choice of craftable food and drinks.

#### Describe the solution
Add many cocktails (some as variants for existing ones), limes, cereal with milk, rote grutze, deep-fried butter and paprika powder, as well as recipes to make those.              

#### Describe alternatives you've considered
None.

#### Testing
Launched the game, made sure that no errors happen.

#### Additional context
None.

### CODE REVIEW

File: data/json/items/comestibles/dairy.json [hunk 1]
```diff
@@ -312,6 +312,49 @@
...
+    "price_postapoc": 150,
+    "material": [ "milk", "wheat" ],
+    "primary_material": "milk",
+    "color": "brown",
+    "spoils_in": "3 days",
+    "volume": "250 ml",
+    "weight": "70 g",
+    "charges": 4,
+    "flags": [ "EATEN_HOT" ],
+    "calories": 413
```
Comment (TheShadowFerret) [inline data/json/items/comestibles/dairy.json:356]:
```suggestion
    "calories": 413,
    "healthy": -1
```

Comment (AtomicFox556) [reply data/json/items/comestibles/dairy.json:356]: I've considering doing that but wasn't sure if it was necessary, as both raw butter and player-made deep-fried food (such deep-fried tripe) lack health penalty. It appears to be limited to alcohol and pre-Cataclysm processed food.

**Comment by Zireael07** [comment]
Why close?

**Comment by sonphantrung** [comment]
@Zireael07 late reply, but the author removed the dda repo (maybe offended by the pride flags addition?), and now working on [a fork](https://github.com/AtomicFox556/Cataclysm-EOD)

**Pull request closed without merging**